defaults/.golangci.yaml

```diff
@@ -1,6 +1,7 @@
 # https://github.com/golangci/golangci/wiki/Configuration
 run:
-  deadline: 3m
```

Comment:
:thisisfinedog:.  I mean, this is fine, despite that linting generally should only take a few seconds.

Reply:
It looks like actions give us very limited resources to run CI, hence the timeouts. We have a more permanent solution down the pipe for linting anyway. This is stopgap.